### PR TITLE
Pull entities functions.

### PR DESCRIPTION
**Pull request by jasebell**

### CODE REVIEW

File: src/kixi/hecuba/weather/service.clj [hunk 1]
```diff
@@ -1,24 +1,50 @@
...
 
 (defn load-config [filename]
   (with-open [r (io/reader filename)]
     (read (java.io.PushbackReader. r))))
 
-(defn find-entities [project-id api-endpoint entity-type]
-  (println project-id)
-  (println api-endpoint)
-  (println entity-type))
+(defn run-api-search [username password project-id api-endpoint entity-type max-entries-per-page]
```
Comment (elisehuard) [inline src/kixi/hecuba/weather/service.clj]: worth starting to use a map and destructuring when faced with large number of args?  Also worth questioning oneself whether they should be in one map or several if subgroups are applicable?

Comment (jasebell) [inline src/kixi/hecuba/weather/service.clj]: That's a good point, that function only started off with args to start off with then got bigger.

Comment (elisehuard) [inline src/kixi/hecuba/weather/service.clj]: is usually how it goes :)

**Comment by elisehuard** [comment]
:shipit:

**Pull request merged**